docs/content/tutorial/customize-cluster.md

```diff
@@ -144,7 +153,10 @@ spec:
         path: ca.crt
```

Comment:
I know you didn't write this, but I find this part a bit confusing... Maybe it can be changed to match the example files from the secret creation section earlier? I think it would look like this? :thinking: 

```yaml
spec:
  customTLSSecret:
    name: hippo.tls
    items:
      - key: tls.crt
        path: hippo.crt
      - key: tls.key
        path: hippo.key
      - key: ca.crt
        path: ca.crt
```